kotlin/internal/toolchains.bzl

```diff
@@ -122,7 +126,7 @@ _kt_toolchain = rule(
         ),
         "language_version": attr.string(
             doc = "this is the -language_version flag [see](https://kotlinlang.org/docs/reference/compatibility.html)",
-            default = "2.0",
+            default = "1.9",
```

Comment:
why are we going back to an older version?

Reply:
I can't merge?